src/xdocs/writingchecks.xml

```diff
@@ -665,7 +665,10 @@ public class MethodLimitCheck extends AbstractCheck
           no
           <a href="https://docs.oracle.com/javase/specs/jls/se8/html/jls-3.html#jls-3.3">
             Unicode escape sequence </a>
-          support (i.e. <code>\uXXXX</code>) in keywords and identifiers.
+          support (i.e. <code>\uXXXX</code>) in keywords and identifiers. Javadoc
```

Comment:
Move "Javadoc" to new line. Please add link https://github.com/checkstyle/checkstyle/issues/10629 until what time it is limitation.
It is not permanent limitation, it is unfortunate lack of time, but we should support it .

Reply:
Done.